app/test_split_integer.py

```diff
@@ -2,20 +2,31 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    # Перевірка, чи сума частин дорівнює значенню
+    assert sum(split_integer(17, 4)) == 17
+    assert sum(split_integer(32, 6)) == 32
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    # Перевірка випадку, коли значення можна рівно поділити на частини
+    assert split_integer(6, 2) == [3, 3]
+    assert split_integer(10, 5) == [2, 2, 2, 2, 2]
 
 
 def test_should_return_part_equals_to_value_when_split_into_one_part() -> None:
-    pass
+    # Перевірка випадку, коли значення ділиться на одну частину
+    assert split_integer(8, 1) == [8]
+    assert split_integer(100, 1) == [100]
 
 
 def test_parts_should_be_sorted_when_they_are_not_equal() -> None:
-    pass
+    # Перевірка, чи частини відсортовані за зростанням
+    result = split_integer(17, 4)
+    assert result == [4, 4, 4, 5]
+    assert result == sorted(result)  # Перевіряємо, чи відсортовані частини
```

Comment:
The assertion `assert result == sorted(result)` is redundant if the task does not explicitly require the parts to be sorted. Since the `split_integer` function already sorts the parts before returning, this check may not be necessary unless you want to ensure that the function's output is sorted.